### DISCUSSION TITLE
A couple of TSML UI questions

### DISCUSSION THREAD

**Discussion by toddjsaunders** [code4recovery/12-step-meeting-list#668] [Q&A] [answered]
I'm configuring this and really like the new UI, I have two questions that have come up --

With the new front end, our online meetings say both "Online" and "Online Meeting" but on the old front end they only say "Online"

![Screen Shot 2022-02-03 at 6 06 41 PM](https://user-images.githubusercontent.com/8978087/152557849-2e82ecb0-512b-4c4b-a27d-a6a4f56c260d.png)

![Screen Shot 2022-02-03 at 6 06 30 PM](https://user-images.githubusercontent.com/8978087/152557947-9fc7601a-9587-413f-a2f4-90dd3211444f.png)

I'd like to remove "Online Meeting" from showing up...

It also shows up in the Type search:

<img width="210" alt="Screen Shot 2022-02-04 at 7 41 27 AM" src="https://user-images.githubusercontent.com/8978087/152558156-01b18024-a3d8-43fc-82bf-865101047c71.png">

Also, on the in-person meetings is there a way to not show the list of other meetings at the same location, and instead just the count, as the old legacy TSML did?

<img width="324" alt="Screen Shot 2022-02-04 at 7 42 55 AM" src="https://user-images.githubusercontent.com/8978087/152558433-438048d3-489a-4d1a-beaa-75c2df3474f4.png">

Thank you!

**Comment by joshreisner** [comment, marked as answer]
hi @toddjsaunders , i've created an issue to fix the extra "Online Meeting" issue: https://github.com/code4recovery/12-step-meeting-list/issues/670

regarding the meeting list, TSML (the WP plugin) does say the number of meetings on the meeting detail page, but that links you to another page for that location where you can see links to all the meetings that are held there. TSML UI on the other hand doesn't have that view, and we needed a place to put that list. it is important for some users to see that, for example users with limited mobility.

what's the reason for wanting to hide it? for example is it that the meetings might not all be affiliated, or is it more of a visual preference?

**Reply by toddjsaunders** [reply]
Hi Josh,

Thanks so much for your reply --

You hit the nail on the head -- not all of the meetings that show up on the meeting detail page are affiliated.

I actually was able to fix the extra "Online Meeting" type showing up -- When I downloaded all our meetings in csv format, I could see the type "Online Meeting" showing up on those meetings that had both types showing in the UI, but not in the legacy TSML. Removing those from the csv and choosing to delete everything during the upload fixed it.

Thanks for the help, keep me posted if hiding the meeting list on the detail page becomes an option.

Best,

Todd

**Reply by joshreisner** [reply]
Ok great, I'm glad the Online Meeting issue is fixed. I also pushed a change this morning to the code which may have helped.

I think we will keep the location listing as-is for now. Like I said, it's useful for users to know which other meetings meet at a given location, for example in the case of someone who doesn't have a car. 

However you can add a second list of only the group meetings, you can do this by adding a group name to your listings. That might allay group concerns about affiliation. See this page as an example: https://aasanjose.org/meetings?meeting=candlelight-group

The first list shows those meetings at the church, the second those of the group.

**Reply by toddjsaunders** [reply]
Hi Josh,

Thanks for the suggestion to add a group name -- is there any way to reverse the display order, so that the group list appears above the location (in your example 'church') list?

Probably more important, I notice that the "Request a change to this listing" button gets pushed to the bottom of the page, below all of the listed meetings, where it's really not visible to the user, is there a way to move it so it appears above the list, just below the meeting information?

I do get your point about it being useful for users to know which other meetings meet at a given location, but I found the "## other meetings at this location" link in the legacy TSML quite logical and intuitive, since not all users care to see that information, and if they do, it's only a click away :-)

Thanks!
Todd

**Reply by joshreisner** [reply]
No, I don't think there's a way to reverse the display order 😞. The boxes are arranged by importance to typical users (meeting > location > group). 

The Update Meeting Info button is also a lower priority than directions and meeting information for most users, so that information still needs to come first. But in a future update that button will move to the area below the map, and there it will be more visually prominent.
